terraform/storage.tf

```diff
@@ -26,8 +56,9 @@ resource "aws_db_instance" "kubeflow_db" {
   ### Auth
   ### ---
   # TODO: randomize or secure this somehow
-  username = "kubeflow"
-  password = "kubeflow" #tfsec:ignore:GEN003
+  username                            = "kubeflow"
+  password                            = "kubeflow" #tfsec:ignore:GEN003
```

Comment:
Local exec provisioner to run an atomic password changing script via a null resource that depends on this `kubeflow_db` resource is I think the _cleanest_ way.